Sources/Positionless/Algorithms/Rotate.swift

```diff
@@ -1,4 +1,4 @@
-extension MutableCollectionPartition {
+extension MutablePartition {
 
   /// Mutates the bisection so that after operation `parts[0]` contains elements
```

Comment:
Unable to make a suggestion that rewrites the whole comment; thanks GitHub.  Eliminating needless words, making the summary a single fragment per API guidelines.  See if you can follow this example elsewhere.

```swift
  /// Exchanges the elements of `parts[0]` with those of `parts[1]` without
  ///  reordering elements within each part.
  ///
  /// - Precondition: `partitionCount == 2`.
  /// - Complexity: O(n) where `n == parts[0].count + parts[1].count`.
```
  
Don't we have a `count` on the whole partitioning?  Seems we should.  Then you can say

```swift
  /// - Complexity: O(`count`).
```

Arguably we should be saying something like “no more than O(`count`) `swap`s” or even better “no more than `2 * count` `swap`s” (or whatever the correct formula is.

Reply:
Resolving this conversation for now. If you think, there is still ambiguity in this, please unresolve it.